delius-test/sub-projects/new-tech.tfvars

```diff
@@ -61,7 +61,9 @@ offender_api_allowed_secure_cidrs = [
   "3.8.51.207/32",     # cloudplatform-live1-2
   "35.177.252.54/32",  # cloudplatform-live1-3
   "35.177.252.195/32", # healthkick
-  "51.140.222.8/32",   # azure oauth server
+  "51.140.222.8/32",   # azure hmpps-auth legacy server
+  "51.11.121.230/32",  # azure hmpps-auth nomisapi-t3 
+  "51.11.124.2/32",    # azure hmpps-auth nomisapi-t2
```

Comment:
interesting about T2 we don't really have an equivalent environment, something to think about

Reply:
Yes, thought I'd add it for completeness, maybe a job for future to fix up t2.